.github/workflows/build-docker-image.yml

```diff
@@ -15,14 +15,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
```

Comment:
Please make these changes in separate PRs

Reply:
Could I start doing it from the next time please? I just want to have it passed and in operation.